app/gui2/src/components/GraphEditor/GraphVisualization.vue

```diff
@@ -46,6 +46,7 @@ const props = defineProps<{
 }>()
 const emit = defineEmits<{
   'update:rect': [rect: Rect | undefined]
+  'update:userSetSize': [size: Vec2]
```

Comment:
You will have two refs either way. This isn't really bidirectional, since you indentionally can't set that size from the GraphNode component, it is effectively read-only. Thus only an emit, but not prop. v-model would be both.